spec/models/efile_submission_spec.rb

```diff
@@ -445,14 +445,15 @@
   end
 
   describe "#retry_send_submission" do
-    context "when the submission is fairly recent" do
-      before do
```

Comment:
The `before do` is not really removed; it's moved higher-up into the `describe ...` out of each `context`.